src/main/java/com/solvdLaba/airport/Flight.java

```diff
@@ -1,101 +1,106 @@
+import java.util.List;
 
 public class Flight {
     private String flightNumber;
     private Airport departureAirport;
     private Airport arrivalAirport;
     private int numberOfSeats;
     protected int availableSeats;
-    Airplane airplane;
     private Date date;
+    private String duration;
+    Airplane airplane;
+    private List<Passenger> passengers = new ArrayList<>();
+    private FlightState flightState;
 
-    public void setFlightNumber(String flightNumber) {
+    public Flight(String flightNumber, Airport departureAirport, Airport arrivalAirport, int numberOfSeats, Date date, String duration) {
         this.flightNumber = flightNumber;
-    }
-
-    public void setDepartureAirport(Airport departureAirport) {
         this.departureAirport = departureAirport;
-    }
-
-    public void setArrivalAirport(Airport arrivalAirport) {
         this.arrivalAirport = arrivalAirport;
-    }
-
-    public void setNumberOfSeats(int numberOfSeats) {
         this.numberOfSeats = numberOfSeats;
+        this.availableSeats = numberOfSeats;
+        this.date = date;
+        this.duration = duration;
     }
-
-    public void setAvailableSeats(int availableSeats) {
-        this.availableSeats = availableSeats;
+    public void addPassenger(Passenger passenger){
+        passengers.add(passenger);
     }
-
-    public void setAirplane(Airplane airplane) {
-        this.airplane = airplane;
+    @Override
+    public String toString(){
+        return "Flight Information\nFlight number: "+flightNumber+"\nDeparture from: "+getDepartureAirport()
+                + "\nArrive to : "+getArrivalAirport() +"\nNumber of seats "+ numberOfSeats+" seats"
+                +"\nAvailable seats are : "+availableSeats + " \nDate: " + date + "\nDuration: " + duration;
+    }
+    public String getFlightNumber() {
+        return flightNumber;
     }
 
-    public void setDate(Date date) {
-        this.date = date;
+    public Airport getDepartureAirport() {
+        return departureAirport;
     }
 
-    public void setDuration(String duration) {
-        this.duration = duration;
+    public Airport getArrivalAirport() {
+        return arrivalAirport;
     }
 
-    public int getAvailableSeats() {
+    public int getAvailableSeats(String preferredClass) {
         return availableSeats;
     }
 
-    public Airplane getAirplane() {
-        return airplane;
-    }
+    public Date getDate() {return date;}
 
-    public Date getDate() {
-        return date;
-    }
+    public int getAvailableSeats() {return availableSeats;}
 
-    public String getDuration() {
-        return duration;
+    public int getNumberOfSeats() {
+        return numberOfSeats;
     }
 
-    private String duration;
+    public String getDuration() {return duration;}
 
-    public Flight(String flightNumber, Airport departureAirport, Airport arrivalAirport, int numberOfSeats, Date date, String duration) {
+    public Airplane getAirplane() {return airplane;}
```

Comment:
format this

Reply:
If I understood correctly I tried to fix the format of the class structure. I hope I did the thing what you requested